doc/API/centreon-api-v2.yaml

```diff
@@ -1265,50 +1389,38 @@ components:
              * `2` - CRITICAL
              * `3` - UNKNOWN
           example: 1
-    Acknowledgment.Service:
+    Acknowledgement.Service:
       allOf:
-        - $ref: '#/components/schemas/Acknowledgment.Host'
+        - $ref: '#/components/schemas/Acknowledgement.Host'
         - type: object
           properties:
             service_id:
               type: integer
               format: int64
               description: "ID of the service"
               example: 5
-    Acknowledgment.Host.Add:
+    Acknowledgement.Host.Add:
       type: object
       properties:
         comment:
           type: string
-          description: "Short description of the acknowledgment"
+          description: "Short description of the acknowledgement"
           example: "Acknowledged by admin"
-        host_id:
-          type: integer
-          format: int64
-          description: "ID of the host"
-          example: 12
         is_notify_contacts:
           type: boolean
           description: "Indicates whether notification is sent to the contacts linked to the host or service"
           example: false
         is_persistent_comment:
           type: boolean
-          description: "Indicates whether acknowledgment is maintained in the case of a restart of the scheduler"
+          description: "Indicates whether acknowledgement is maintained in the case of a restart of the scheduler"
           example: true
         is_sticky:
           type: boolean
-          description: "Indicates whether acknowledgment is maintained in the case of a change of status"
+          description: "Indicates whether acknowledgement is maintained in the case of a change of status"
           example: true
-    Acknowledgment.Service.Add:
+    Acknowledgement.Service.Add:
       allOf:
-        - $ref: '#/components/schemas/Acknowledgment.Host.Add'
-        - type: object
-          properties:
-            service_id:
```

Comment:
shouldn't the serviceId be specified ?

Reply:
it is specified in the URI now : /monitoring/hosts/{hostId}/services/{serviceId}/acknowledgements